Makefile

```diff
@@ -249,25 +249,6 @@ version-dist: nodeup-dist kops-dist protokube-export utils-dist
 	cp ${DIST}/linux/amd64/utils.tar.gz ${UPLOAD}/kops/${VERSION}/linux/amd64/utils.tar.gz
 	cp ${DIST}/linux/amd64/utils.tar.gz.sha1 ${UPLOAD}/kops/${VERSION}/linux/amd64/utils.tar.gz.sha1
 
-.PHONY: vsphere-version-dist
-vsphere-version-dist: nodeup-dist protokube-export
```

Comment:
I am not certain how this works, and why it is in here, we need one of the folks from VMware to comment.